int64 ping
---
int64 res
